### PR TITLE
[FIVE-275]Reducción de bundle sin uso

### PR DESCRIPTION
**Pull request by gonzalorom**
#### Error: heap out of memory.

- Se eliminaron paquetes en desuso con el fin de disminuir dependencias y reducir el tamaño de la app.

### CODE REVIEW

File: FiveRockingFingers/FRF.Web/ClientApp/package.json [hunk 1]
```diff
@@ -3,55 +3,52 @@
...
-    "popper.js": "^1.16.0",
-    "react": "16.11.0",
-    "react-dom": "16.11.0",
-    "react-hook-form": "^6.9.2",
-    "react-moment": "^1.0.0",
-    "react-pro-sidebar": "^0.4.4",
-    "react-router": "5.1.2",
+    "merge": "2.1.0",
+    "popper.js": "^1.16.1",
+    "react": "17.0.1",
```
Comment (matiasbeckerle) [inline FiveRockingFingers/FRF.Web/ClientApp/package.json:20]: You're also making quite big upgrades on some libraries. Please have in mind avoid doing that. Upgrading a library deserves a single standalone PR by itself, specially a key one like React. If you want to update some of these, please consider upgrading in small chunks in order to monitor that everything still works.

In a real live production project this is a no-go, very very risky.

File: FiveRockingFingers/FRF.Web/ClientApp/package.json [hunk 2]
```diff
@@ -61,7 +58,8 @@
     "build": "react-scripts build",
     "test": "cross-env CI=true react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "lint": "eslint ./src/**/*.ts ./src/**/*.tsx"
+    "lint": "eslint ./src/**/*.ts ./src/**/*.tsx",
+    "analyse": "node analyse.js"
```
Comment (matiasbeckerle) [inline FiveRockingFingers/FRF.Web/ClientApp/package.json:62]: What it is this for?

File: FiveRockingFingers/FRF.Web/ClientApp/package.json [hunk 3]
```diff
@@ -3,55 +3,52 @@
...
+    "@typescript-eslint/parser": "^4.12.0",
+    "cross-env": "7.0.3",
+    "eslint": "^7.17.0",
+    "eslint-plugin-flowtype": "^5.2.0",
+    "eslint-plugin-import": "2.22.1",
+    "eslint-plugin-jsx-a11y": "6.4.1",
+    "eslint-plugin-react": "7.22.0",
+    "nan": "^2.14.2",
+    "typescript": "^4.1.3",
+    "webpack-bundle-analyzer": "^4.3.0"
```
Comment (matiasbeckerle) [inline FiveRockingFingers/FRF.Web/ClientApp/package.json:51]: You need this one?

File: FiveRockingFingers/FRF.Web/ClientApp/package.json (hunk 1 above)
Comment (gonzalorom) [reply FiveRockingFingers/FRF.Web/ClientApp/package.json:20]: I have read the documentation, and it is safe to make these updates, but I understand what you are saying. Will rollback all the versions and bring these changes on the table for the future. Thank you.

Comment (matiasbeckerle) [reply FiveRockingFingers/FRF.Web/ClientApp/package.json:20]: Awesome, glad you checked changelog for breaking compatibility. Don't throw away that work: add a follow up PR with that change. That way if anything happens it will be easier to recognize where the app got to a broken state by reviewing the commits history, an always useful strategy. Plus, you practice for production-ready projects :)

File: FiveRockingFingers/FRF.Web/ClientApp/package.json (hunk 2 above)
Comment (gonzalorom) [reply FiveRockingFingers/FRF.Web/ClientApp/package.json]: Done.

File: FiveRockingFingers/FRF.Web/ClientApp/package.json (hunk 3 above)
Comment (gonzalorom) [reply FiveRockingFingers/FRF.Web/ClientApp/package.json]: Done.

**Review by matiasbeckerle — APPROVED** [review]
Thanks for always considering the feedback. Looks great to me!

File: FiveRockingFingers/FRF.Web/ClientApp/package.json (hunk 1 above)
Comment (gonzalorom) [reply FiveRockingFingers/FRF.Web/ClientApp/package.json]: Now we have all these changes in a new PR, in order to achieve what you advised.

**Comment by gonzalorom** [comment]
Thank you, for all the feedback. I'm always looking forward to the code review!
